R/LongitudinalModel.R

```diff
@@ -51,3 +51,34 @@ as_print_string.LongitudinalModel <- function(object, ...) {
     )
     return(string)
 }
+
+
+#' @export
+linkTTG.LongitudinalModel <- function(object, ...) {
```

Comment:
this works well for the packaged links, I wonder if this could be positioned "higher" in the methods tree, such that when the user / externally defined link is called without the method implementation it would also give a sensible error message?

Reply:
Yes, exactly - what happens if the user defines a `link_xyz()` function and then there is no method for object `GSF`? Is that caught? (I guess not)